mongodb/pom.xml

```diff
@@ -66,6 +80,17 @@
             <artifactId>parboiled-java</artifactId>
         </dependency>
         <dependency>
+            <groupId>com.github.fakemongo</groupId>
+            <artifactId>fongo</artifactId>
+            <optional>true</optional>
+        </dependency>
+        <dependency>
+            <groupId>org.slf4j</groupId>
+            <artifactId>slf4j-simple</artifactId>
+            <version>${slf4jVersion}</version>
+            <scope>provided</scope>
```

Comment:
I feel this should rather be marked as optional. "Provided" suggests its needed at runtime (just provided by the environment), but that's not the case for the normal MongoDB case. I'd mark both deps as optional and add a short comment here. WDYT?